src/EA.Weee.RequestHandlers/Admin/Reports/GetAatfAeReturnDataCsvHandler.cs

```diff
@@ -37,6 +37,11 @@ public async Task<CSVFileData> HandleAsync(GetAatfAeReturnDataCsv request)
                        request.ComplianceYear, request.Quarter, (int)request.FacilityType, 
                        request.ReturnStatus, request.AuthorityId, request.LocalArea, request.PanArea);
 
+            foreach (var item in items)
+            {
+                 item.AatfDataUrl = string.Format(@" =HYPERLINK(""""{0}{1}#data"""", """"View AATF data"""")", request.AatfDataUrl, item.AatfId);
```

Comment:
Just add test to check that this hyperlink is set correctly

Reply:
Added